### PR TITLE
DSD-494: expose Chakra Grid component

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-494](https://jira.nypl.org/browse/DSD-494)

## This PR does the following:
- exports Chakra's native `Grid` component.

## How has this been tested?

- Storybook in local DS build

### Checklist:

- [x] I have updated the Storybook documentation accordingly.
- [x] All new and existing tests passed.

### Front End Review:

- [x] View [the example in Storybook](https://pr698-nz58ovdmwng57yrew7iqkftycvusnkfm.tugboat.qa/)

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr698-nz58ovdmwng57yrew7iqkftycvusnkfm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/615345d34807c085313d67c3

File: src/components/Chakra/Grid.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { Box } from "@chakra-ui/react";
+import { Grid, GridItem } from "@chakra-ui/react";
+
+import { getCategory } from "../../utils/componentCategories";
+import VideoPlayer from "../VideoPlayer/VideoPlayer";
```
Comment (EdwinGuzman) [inline src/components/Chakra/Grid.stories.mdx]: I think these were accidentally added -- they're not used in this file.

**Comment by EdwinGuzman** [comment]
This is fine but you mentioned `GridItem` in the docs so I think it should also be exported from the DS.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr698-nz58ovdmwng57yrew7iqkftycvusnkfm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/615345d34807c085313d67c3

File: src/components/Chakra/Grid.stories.mdx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Chakra/Grid.stories.mdx]: Ugh.  Leftover from the file I initially copied.  Now removed.

**Comment by bigfishdesign13** [comment]
> This is fine but you mentioned `GridItem` in the docs so I think it should also be exported from the DS.

@EdwinGuzman Yes!  I meant to do that.  Good catch.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr698-nz58ovdmwng57yrew7iqkftycvusnkfm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/615345d34807c085313d67c3

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**